README.md

```diff
@@ -180,6 +181,25 @@ Additionally, container image used to deliver this plugin can be set using
 `KUBE_SECONDARY_DNS_IMAGE` environment variable in operator
 deployment manifest.
 
+## kubevirtIpamController
+
+[This controller](https://github.com/maiqueb/kubevirt-ipam-claims)
+allows to support IPAM for secondary networks.
```

Comment:
We are going to have IPAM for primary networks so let's rename secondary with user defined
```suggestion
allows to support IPAM for user defined networks.
```